pay-api/devops/vaults.json

```diff
@@ -34,7 +34,8 @@
         "application": [
             "postgres-pay",
             "pay-api",
-            "jwt"
+            "jwt",
+            "account-mailer"
```

Comment:
Wondering if there is any overlaps of secrets between account-mailer and pay-api... also probably better to just add it into `pay-api`